test_ngi_config.yaml

```diff
@@ -0,0 +1,97 @@
+# This file should be placed under $HOME/.ngiconfig/ngi_config.yaml
+# or its path exported as the environment variable NGI_CONFIG
+
+database:
+    # SQLite file to know where things are happening (state machine to back up Charon for network failure)
+    # -usually it is at /proj/a2014205/ngi_resources/record_tracking_database_nestor.sql
+    # -if you are trying to work without one, it is going to create an empty schema
+    record_tracking_db_path: /home/szilva/glob/private/sb/tracking/record_tracking_database.sql
```

Comment:
The the scripts points to a combination of nestor resources as well as @szilvajuhos personal folders. I like the comments, but I don't like the default values for the variables.

We could probably put this into an ansible template. We realised we didn't know where people would dev ngi_pipeline in the future though, so we put the idea on hold.